docs/next/modules/en/pages/troubleshooting/troubleshooting.adoc

```diff
@@ -0,0 +1,300 @@
+| *-controller-manager
+| Handles the initialization and management of Infrastructure provider components
+
+| $-bootstrap-system
+| $-bootstrap-controller-manager
+| Manages the bootstrap configuration for custom providers
+
+| $-control-plane-system
+| $-control-plane-controller-manager
+| Handles the initialization and management of custom providers control plane components
+|===
+
+'*' - depending on the infrastructure provider in use
+'$' - depending on controlplane/bootstrap provider in use, e.g. kubeadm or rke2
+
+One way to monitor logs from Cluster API controll plane pods by usuing `kubectl logs` with label selector.
+
+[source,bash]
+----
+kubectl logs -l control-plane=controller-manager -n rancher-turtles-system -f
+kubectl logs -l control-plane=controller-manager -n capi-system -f
+kubectl logs -l control-plane=controller-manager -n rke2-bootstrap-system -f
+kubectl logs -l control-plane=controller-manager -n rke2-control-plane-system -f
+----
+
+[CAUTION]
+====
+Using kubectl krew to install the stern plugin involves third-party software which may pose security risks in production environments. While it's a useful tool for log viewing, it should primarily be used in development or testing environments. Always review third-party plugins before installing them in sensitive or production clusters.
+====
+
+The best method to monitor logs from different pods is by using the following method.
+
+[source,bash]
+----
+kubectl krew install stern
+----
+
+Then you can tail logs from all pods in real time.
+
+[source,bash]
+----
+kubectl stern . -n rancher-turtles-system -n rke2-bootstrap-system -n rke2-control-plane-system -n capi-system
+----
+
+or
+
+[source,bash]
+----
+kubectl stern -A -l control-plane=controller-manager
+----
+
+== How to manage Rancher Turtles and CAPI resources
+
+*List Rancher and Cluster API resources*
+
+Often users are asking how to deal with long list of different CRDs created by CAPI and Rancher, here is some idea how you can deal with it.
+
+. List all CAPI and Rancher related credentials
++
+[source,bash]
+----
+kubectl api-resources | grep 'cattle.io\|cluster.x-k8s.io'
+----
++
+. About the resource, you can get a full explanation of what it is doing or what the configuration options are
++
+[source,bash]
+----
+kubectl api-resources | grep clusterclass
+kubectl explain clusterclass.cluster.x-k8s.io
+----
++
+. Then you get a description about each field and configuration section 
++
+[source,bash]
+----
+kubectl explain clusterclass.spec
+kubectl explain clusterclass.spec.controlPlane
+kubectl explain clusterclasses.spec.controlPlane.machineInfrastructure
+----
+
+*List existing Rancher and Cluster API objects resources*
+
+The most common mistake is to use the `default` namespace in Kubernetes for your work. This will make your configuration really messy and difficult to manage and troubleshoot. It is highly recommended to use separate namespaces for deploying and creating CAPI downstream clusters.
```

Comment:
```suggestion
The most common mistake is to use the `default` namespace in Kubernetes for your work. This will make your configuration really messy and difficult to manage and troubleshoot. It is highly recommended to use separate namespaces for deploying and creating CAPI downstream clusters, and avoid using kubernetes system namespaces, such as `default` and `kube-*`.

When applying templates with `kubectl`, always specify `-n <namespace>` to provision resources in a known location.
```

Reply:
added